Framework/Core/src/ExternalFairMQDeviceProxy.cxx

```diff
@@ -407,10 +408,13 @@ void injectMissingData(fair::mq::Device& device, fair::mq::Parts& parts, std::ve
       parts.AddPart(std::move(headerMessage));
       // add empty payload message
       parts.AddPart(device.NewMessageFor(channelName, 0, 0));
+      if (concrete.origin != o2::header::gDataOriginEMC || concrete.origin != o2::header::gDataOriginPHS || concrete.origin != o2::header::gDataOriginHMP || concrete.dataDescription != o2::header::DataDescription{"RAWDATA"}) {
```

Comment:
Shouldn't this be
```
if ((concrete.origin != o2::header::gDataOriginEMC && concrete.origin != o2::header::gDataOriginPHS && concrete.origin != o2::header::gDataOriginHMP) || concrete.dataDescription != o2::header::DataDescription{"RAWDATA"})
```
to exclude all origins that are iether EMC, PHS or HMP?

Reply:
yes, will fix online.